charts/newrelic-infrastructure-v3/templates/control-plane/_helpers.tpl

```diff
@@ -0,0 +1,8 @@
+{{/* Returns whether the controlPlane component needs needsHostNetwork to work, that is, if they do not have a staticEndpoint configured */}}
```

Comment:
"if they do not have a staticEndpoint configured"

A nit, the description seems to suggest what we are going to check, but is it not what we check.

Moreover They should not have "a" staticEndpoint, but all static endpoints for the enabled components